src/datasets/utils/resources/languages.json

```diff
@@ -58,6 +58,7 @@
     "ar-TD": "Arabic (Chad)",
     "ar-TN": "Arabic (Tunisia)",
     "ar-YE": "Arabic (Yemen)",
+    "ara":"Arabic",
```

Comment:
When there exist several codes (ISO 639 1, 2, 3) for the same language, we just use the lower one. For Arabic, we use `ar`.

As `ar` already appears in the README, I would remove `ara` from the README and would not add it to `languages.json`

```suggestion
```